anda/devs/zed-nightly/zed-nightly.spec

```diff
@@ -0,0 +1,95 @@
+%global commit 0129d4e2506d5ec5e50ef0968382770b9abec390
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+%global commit_date 20240619
+%global ver v0.142.0
```

Comment:
Strip the `v` prefix as according to update.rhai

Reply:
Done